index.js

```diff
@@ -52,79 +53,113 @@ const root =
   // clicked on (in terminals such as iTerm) to open the file in an editor.
   `${process.cwd()}/`
 
-expandThunks = (array) => array.map(
-  elem => typeof elem == 'function' ? elem() : elem
-)
+const PRETTY = Boolean(process.env.PRETTY)
+const DEPTH = process.env.DEPTH
```

Comment:
does this need to be defaulted?

Reply:
nope, its defaulted in `stringify` function